libs/cardano-ledger-api/src/Cardano/Ledger/Api/State/Query.hs

```diff
@@ -243,3 +247,8 @@ queryCommitteeMembersState coldCredsFilter hotCredsFilter statusFilter nes = do
       , csQuorum = comQuorum
       , csEpochNo = currentEpoch
       }
+
+queryAccountState ::
+  NewEpochState era ->
+  AccountState
+queryAccountState = view $ nesEsL . esAccountStateL
```

Comment:
Is this the same as `n ^. nesEsL . esAccountStateL` ?

Reply:
Yeah, `^.` is `flip view` iirc